app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -41,9 +41,9 @@ This can help when you have (one or more of the following):
 
 ## How to do an SKE course
 
-When you apply for teacher training in England, talk to your training provider if you think you need to improve your subject knowledge. They’ll help you decide if you need to complete an SKE course, and may make it a condition of their offer to train with them.
+If you think you need to improve your subject knowledge then talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from a SKE course. You can only complete a funded SKE course if your training provider makes it a condition of the offer to train with them.
```

Comment:
```suggestion
If you think you need to improve your subject knowledge, talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from a SKE course. You can only complete a funded SKE course if your training provider makes it a condition of the offer to train with them.
```

Reply:
@philspray I don't know whether `when you apply for teacher training in England` is quite right - wouldn't it be before you apply? However, I appreciate this is what is says in production at the moment!

Worth clarifying with the SKE team

Reply:
Would it be worth splitting `You can only complete......` onto a new paragraph? Feels like a lot of info to take in here

Reply:
I'm not sure `help YOU to decide` is right - it is the provider who makes SKE a condition of offer, not you choosing whether to do it or not? Worth clarifying with the SKE team